app/javascript/components/student/IterationPage.tsx

```diff
@@ -66,6 +67,13 @@ export const IterationPage = ({
     }
   }, [CACHE_KEY, solutionId])
 
+  /* Only run this the first time that the component loads */
```

Comment:
Isn't this the definition of what `useEffect` does @kntsoriano ? If so, does the comment add much value?

Reply:
(This is my code)

I wanted to make it clear that we're only wanting to set the initial number of iterations in this starting state, rather than whenever resolvedData updates. But if this is the "normal" usage of `useEffect`, then we should get rid of the comment :)

Reply:
I'm not entirely sure, which is why I asked @kntsoriano (I knew it was your code :))